docs/external-contracts/external-contracts.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+---
+
+# 🛠 Interacting with External Contracts
```

Comment:
How about any of these? 🌐 📡 🔗

Reply:
Ohh yeah nice!, used 📡  at https://github.com/scaffold-eth/se2-docs/pull/74/commits/434b53a12dbf0818b04c54ed6e6429951cdb36e8